yotiprofile.go

```diff
@@ -164,12 +124,65 @@ func (p Profile) DocumentImages() (*attribute.ImageSliceAttribute, error) {
 	return nil, nil
 }
 
+// ApplicationName is the name of the application
+func (p ApplicationProfile) ApplicationName() *attribute.StringAttribute {
+	return p.GetStringAttribute(attrConstApplicationName)
+}
+
+// ApplicationURL is the URL where the application is available at
+func (p ApplicationProfile) ApplicationURL() *attribute.StringAttribute {
+	return p.GetStringAttribute(attrConstApplicationURL)
+}
+
+// ApplicationReceiptBgColor is the background colour that will be displayed on
+// each receipt the user gets as a result of a sharing with the application.
+func (p ApplicationProfile) ApplicationReceiptBgColor() *attribute.StringAttribute {
+	return p.GetStringAttribute(attrConstApplicationReceiptBGColor)
+}
+
+// ApplicationLogo is the logo of the application that will be displayed to
+// those users that perform a sharing with it.
+func (p ApplicationProfile) ApplicationLogo() *attribute.ImageAttribute {
+	return p.GetImageAttribute(attrConstApplicationLogo)
+}
+
 // GetAttribute retrieve an attribute by name on the Yoti profile. Will return nil if attribute is not present.
-func (p Profile) GetAttribute(attributeName string) *attribute.GenericAttribute {
+func (p baseProfile) GetAttribute(attributeName string) *attribute.GenericAttribute {
```

Comment:
I _think_ we'll have to think of exposing this method another way - since the previous version was exposed on the profile object, and we're now extending an unexposed struct, it would be technically removing this method from a client's perspective, and so would count as a breaking change.

Reply:
GetAttribute is still publically exposed on Profile I think? The tests in yoti_test.go still use it.

Reply:
Yes sorry, you're right, it's still exposed - I just checked by using it from the profile example (I think the tests are in the same package, so it wouldn't need to be exposed in order to access them from there)